src/EnergyPlus/ScheduleManager.cc

```diff
@@ -94,9 +94,12 @@ namespace Sched {
     int GetScheduleTypeNum(EnergyPlusData &state, std::string const &name)
     {
         auto const &s_sched = state.dataSched;
-        for (int i = 0; i < (int)s_sched->scheduleTypes.size(); ++i)
-            if (s_sched->scheduleTypes[i]->Name == name) return i;
-        return -1;
+        for (int i = 0; i < (int)s_sched->scheduleTypes.size(); ++i) {
+            if (s_sched->scheduleTypes[i]->Name == name) {
+                return i;
+            }
+        }
+        return SchedNum_Invalid;
```

Comment:
Return the same constexpr int for clarity.

And use braces so we can set a breakpoint if we want to. (we should tell clang-format to always do that IMHO)

Reply:
:+1: Yeah, I'm not a fan of unbraced if statements for nearly any use case.